mmv1/products/networksecurity/AdressGroup.yaml

```diff
@@ -0,0 +1,121 @@
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 30
+      update_minutes: 30
+      delete_minutes: 30
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+autogen_async: true
+import_format:
+  ["projects/{{project}}/locations/{{location}}/addressGroups/{{name}}"]
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: "network_security_address_groups_basic"
+    primary_resource_id: "default"
+    vars:
+      resource_name: "my-address-groups"
+  - !ruby/object:Provider::Terraform::Examples
+    min_version: beta
+    name: "network_security_address_groups_advanced"
+    primary_resource_id: "default"
+    vars:
+      resource_name: "my-address-groups"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    url_param_only: true
+    description: |
+      Name of the AddressGroup resource. It matches pattern projects/*/locations/{location}/addressGroups/<addressGroup>.
+  - !ruby/object:Api::Type::String
+    name: "location"
+    default_value: "global"
+    description: |
+      The location of the gateway security policy.
+      The default value is `global`.
+    url_param_only: true
+properties:
+  - !ruby/object:Api::Type::String
+    name: "description"
+    description: |
+      Free-text description of the resource.
+  - !ruby/object:Api::Type::Time
+    name: "createTime"
+    description: |
+      The timestamp when the resource was created.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z"
+    output: true
+  - !ruby/object:Api::Type::Time
+    name: "updateTime"
+    description: |
+      The timestamp when the resource was updated.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits.
+      Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+    output: true
+  - !ruby/object:Api::Type::KeyValuePairs
+    name: "labels"
+    description: |
+      Set of label tags associated with the AddressGroup resource.
+      An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+  - !ruby/object:Api::Type::Enum
+    name: "type"
+    description: |
+      The type of the Address Group. Possible values are "IPV4" or "IPV6".
+    values:
+      - :IPV4
+      - :IPV6
+  - !ruby/object:Api::Type::Array
+    name: "items"
+    item_type: Api::Type::String
+    required: true
```

Comment:
From the API docs it seems this field is optional - could you please update this in the yaml?

https://cloud.google.com/traffic-director/docs/reference/network-security/rest/v1beta1/organizations.locations.addressGroups#AddressGroup.FIELDS.items